icd/CMakeLists.txt

```diff
@@ -9,8 +9,8 @@ set (PYTHON_CMD ${PYTHON_EXECUTABLE})
 # Define macro used for building vk.xml generated files
 macro(run_vk_xml_generate dependency output)
     add_custom_command(OUTPUT ${output}
-    COMMAND ${PYTHON_CMD} ${SCRIPTS_DIR}/lvl_genvk.py -registry ${SCRIPTS_DIR}/vk.xml ${output}
-    DEPENDS ${SCRIPTS_DIR}/vk.xml ${SCRIPTS_DIR}/generator.py ${SCRIPTS_DIR}/${dependency} ${SCRIPTS_DIR}/lvl_genvk.py ${SCRIPTS_DIR}/reg.py
+    COMMAND ${PYTHON_CMD} ${SCRIPTS_DIR}/v_t_genvk.py -registry ${SCRIPTS_DIR}/vk.xml ${output}
```

Comment:
Are we sold on v_t_genvk.py, or is vt_genvk.py ok? Reduces required shifts by 50%.

Reply:
I started off that way, but what should we then call the one in VulkanTools?

Reply:
This one could be "kvt_genvk.py" and that one could be "lvt_genvk.py"?

Reply:
OK Tobin, it shall be so.  Though, I have to say this is a lot coming from the creator of ~^~ field separators. 👍

Reply:
I seriously almost qualified my suggestion by saying "just remember this is coming from the creator of the ~^~ delimiter". I know how much everyone loves that!